include/robot_state_publisher/joint_state_listener.h

```diff
@@ -65,15 +65,15 @@ class JointStateListener {
    */
   JointStateListener(const KDL::Tree& tree, const MimicMap& m, const urdf::Model& model = urdf::Model());
 
-  JointStateListener(const std::shared_ptr<RobotStatePublisher>& rsp, const MimicMap& m);
+  JointStateListener(std::shared_ptr<RobotStatePublisher>  rsp, MimicMap  m);
 
 
   /// Destructor
   ~JointStateListener();
 
 protected:
-  virtual void callbackJointState(const JointStateConstPtr& state);
-  virtual void callbackFixedJoint(const ros::TimerEvent& e);
+  void callbackJointState(const JointStateConstPtr& state);
+  void callbackFixedJoint(const ros::TimerEvent& e);
```

Comment:
I believe that these are meant to be virtual, so that derivative classes can properly override them.